common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala

```diff
@@ -275,5 +278,78 @@ object CuratedRecipe {
     })
   }
 
+  def toAtom(r: Recipe, cr: CuratedRecipe): Atom = {
+
+    val contentChangeDetails = ContentChangeDetails(
+      created = Some(
+        ChangeRecord(
+          date = r.publicationDate.toInstant.toEpochMilli,
+          user = Some(
+            User(email = "off-platform@guardian.co.uk")
+          )
+        )
+      ),
+      published = Some(
+        ChangeRecord(
+          date = r.publicationDate.toInstant.toEpochMilli,
+          user = Some(
+            User(email = "off-platform@guardian.co.uk")
+          )
+        )
+      ),
+      revision = 1L
+    )
+
+    val recipeAtom = atom.RecipeAtom(
+      title = cr.title,
+      tags = atom.Tags(
```

Comment:
The issue of exposing `dietary` and `holidays` back to the user is in progress....